<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MirrorDock</class>
 <widget class="QDockWidget" name="MirrorDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&amp;Copy, Mirror and Rotate</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Copy</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="2" column="1">
        <widget class="QToolButton" name="copyDown">
         <property name="toolTip">
          <string>Copy down</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/copy_down.svg</normaloff>:/images/copy_down.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="copyUp">
         <property name="toolTip">
          <string>Copy up</string>
         </property>
         <property name="text">
          <string>U</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/copy_up.svg</normaloff>:/images/copy_up.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="copyRight">
         <property name="toolTip">
          <string>Copy right</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/copy_right.svg</normaloff>:/images/copy_right.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="copyLeft">
         <property name="toolTip">
          <string>Copy left</string>
         </property>
         <property name="text">
          <string>L</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/copy_left.svg</normaloff>:/images/copy_left.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Mirror</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="0" column="1">
        <widget class="QToolButton" name="mirrorUp">
         <property name="toolTip">
          <string>Mirror up</string>
         </property>
         <property name="text">
          <string>U</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_up.svg</normaloff>:/images/mirror_up.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="mirrorDown">
         <property name="toolTip">
          <string>Mirror down</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_down.svg</normaloff>:/images/mirror_down.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="mirrorLeft">
         <property name="toolTip">
          <string>Mirror left</string>
         </property>
         <property name="text">
          <string>L</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_left.svg</normaloff>:/images/mirror_left.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="mirrorRight">
         <property name="toolTip">
          <string>Mirror right</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_right.svg</normaloff>:/images/mirror_right.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Rotate</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="2" column="1">
        <widget class="QToolButton" name="rotate180">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rotate 180&amp;#0176;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>180</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate180.svg</normaloff>:/images/rotate180.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="rotate270">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rotate 270&amp;#0176;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>270</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate270.svg</normaloff>:/images/rotate270.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="rotate90">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rotate 90&amp;#0176;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>90</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate90.svg</normaloff>:/images/rotate90.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="rotateCustom">
         <property name="toolTip">
          <string>Rotate custom degrees</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate_custom.svg</normaloff>:/images/rotate_custom.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="rotateCustomWidgets" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="rotateBttn">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="degrees">
         <property name="maximum">
          <double>1440.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../crochet.qrc"/>
 </resources>
 <connections/>
</ui>
